modules/local/plotvcfstats/meta.yml

```
@@ -0,0 +1,58 @@
+name: plotvcfstats
+description: Plots the output of bcftools stats
+keywords:
+  - visualization
+  - stats
+  - VCF
+tools:
+  - plot:
+      description: |
+        Script for processing output of bcftools stats, plots graphs and creates a PDF presentation.
+      homepage: http://samtools.github.io/bcftools/bcftools.html
+      documentation: http://www.htslib.org/doc/bcftools.html
+      doi: 10.1093/bioinformatics/btp352
+      licence: ["MIT"]
+      identifier: biotools:bcftools
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. [ id:'sample1' ]
+    - stats:
+        type: file
+        description: Text file from BCFtools stats output
+        pattern: "*_{stats.txt}"
+        ontologies: []
+output:
+  plot_dir:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. [ id:'sample1' ]
+      - "*plots*":
+          type: folder
+          description: Output directory containing plots, raw data, and log files
+          pattern: "*plots*"
+          ontologies: []
+  plot_pdf:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. [ id:'sample1' ]
+      - "*.pdf":
+          type: file
+          description: Copy of the PDF summary output
+          pattern: "*.{pdf}"
+          ontologies: []
```

```suggestion
          ontologies:
              - edam: http://edamontology.org/format_3508 # PDF
```

PDF has a format ontology

---

Oh I just realised how to find these ontologies. Thank you so much for the input!